### PR TITLE
Reporting page setup

### PR DESCRIPTION
**Pull request by achiu8**

### CODE REVIEW

File: src/components/Reporting.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  componentWillMount() {
+    this.props.loadOrders();
+  }
+
+  render() {
+    return ReportingNav(
+      {
+        activeTab: this.state.activeTab,
+        setActiveTab: this.setActiveTab
```
Comment (kprakobkit) [inline src/components/Reporting.js]: Why passing setActiveTab as a prop into ReportingNav?

File: src/components/Reporting.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+function mapStateToProps(state) {
+  return {
+    orders: state.orders
+  };
+}
+
+class Reporting extends Component {
+  constructor(props) {
+    super(props);
+    this.setActiveTab = this.setActiveTab.bind(this);
```
Comment (kprakobkit) [inline src/components/Reporting.js:17]: Why define setActiveTab and tracking activeTab in the ReportComponent instead of the ReportingNavComponent?

Comment (achiu8) [inline src/components/Reporting.js:17]: Thinking was that this parent component would also be responsible for rendering the actual page / content components, so it needs to know about the active tab.

Comment (kprakobkit) [inline src/components/Reporting.js:17]: Ah makes sense.

**Pull request merged**